deck.md

```diff
@@ -324,6 +295,69 @@ class: branded
 
 # Step 2: Solution /3
 
+```js
+// routes/users.js
+export default async function users(fastify) {
+  fastify.get('/users', {}, async () => [
+    { username: 'alice' },
+    { username: 'bob' },
+  ])
+}
+```
+
+---
+
+# Step 2: Trying it out
+
+#### Note that the / route is now not found
+
+```a
+curl http://localhost:3000/
+{"message":"Route GET:/ not found","error":"Not Found","statusCode":404}
+```
+
+#### We'll find our response at /users
+
+```a
+curl http://localhost:3000/users
+[{"username":"alice"},{"username":"bob"}]
+```
+
+---
+
+class: branded
+
+# Step 4: Serialization
+
+- Fastify uses a schema-based approach, and even if it is not mandatory we recommend using JSON Schema to validate your routes and serialize your outputs. Internally, Fastify compiles the schema into a highly performant function
+
+- We encourage you to use an output schema, as it can drastically increase throughput and help prevent accidental disclosure of sensitive information
+
+https://www.fastify.io/docs/latest/Validation-and-Serialization/#serialization
+
+---
+
+class: branded
+
+# Step 4: Exercise 💻
+
+- respond to requests with an array of two users:
+
+```json
+[{ "username": "alice" }, { "username": "bob" }]
+```
```

Comment:
I think we are already doing this in step 2, we should remove it from here and place it in step 2